src/profile/templates/user_management/profile/delete-profile-confirmation.html

```diff
@@ -0,0 +1,9 @@
+<div data-extend-template="templates/base.html" class="profile-page">
+    <div data-block="page-body-content-inner">
+        <span translate>Your account has been deleted.</span>
+
+        <div class="home-button">
+            <a class="log-out button right-arrow seperated" ng-href="{{ 'LogoutCtrl'|reverseUrl }}" translate>Home</a>
```

Comment:
Even better add an option to the [userManagementProfileConfig](https://github.com/incuna/angular-user-management/blob/delete-function/src/profile/scripts/providers.js#L12) provider and use that.